sdks/python/container/boot.go

```diff
@@ -211,7 +211,7 @@ func main() {
 	for _, workerId := range workerIds {
 		go func(workerId string) {
 			log.Printf("Executing: python %v", strings.Join(args, " "))
-			log.Fatalf("Python exited: %v", execx.ExecuteEnv(map[string]string{"WORKER_ID": workerId}, "python", args...))
+			log.Printf("Python exited: %v", execx.ExecuteEnv(map[string]string{"WORKER_ID": workerId}, "python", args...))
 			wg.Done()
```

Comment:
```suggestion
			wg.Done()
			log.Fatalf("Python exited: %v", execx.ExecuteEnv(map[string]string{"WORKER_ID": workerId}, "python", args...))
```

Instead of downgrading this, we should probably just move it behind `wg.Done()`. This does still represent a failure case

Reply:
I'll note that the wait group state won't matter because log.Fatal will kill the binary anyway. 

Sibling worker processes (as python uses) need everything to be torn down when one of them dies and exits out. This keeps cleaner parity with the other SDK languages, and matches execution model assumptions. (an SDK worker dying means there's a problem with the VM generally, and it's better for the runner to start the whole thing fresh).

Reply:
Yeah, that's a good point - so probably it can just be

```suggestion
			log.Fatalf("Python exited: %v", execx.ExecuteEnv(map[string]string{"WORKER_ID": workerId}, "python", args...))
```

with no teardown